cmd/puma-dev/main_darwin.go

```diff
@@ -34,6 +34,8 @@ var (
 
 	fCleanup   = flag.Bool("cleanup", false, "Cleanup old system settings")
 	fUninstall = flag.Bool("uninstall", false, "Uninstall puma-dev as a user service")
+
+	shutdown = make(chan os.Signal, 1)
```

Comment:
expose this to the `main` package, so we can grab it from tests and shutdown our goroutine'd server nicely.